common/config_parser/config_parser.cpp

```diff
@@ -68,7 +73,8 @@ ExpectedBool MenderConfigFromFile::ValidateServerConfig() const {
 
 	if (servers.size() == 0) {
 		if (server_url.size() == 0) {
-			// TODO - Log warning
```

Comment:
Lol, who thought it was too much work to add this the first time around? 😂

Reply:
Did we even have logging at that time?

Reply:
Looks like it, but they were done in parallel